app.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+
+//require needed dependencies
```

Comment:
Why this comment only here? If you'd like to separate your files in sections with such comments do it everywhere.